postixdroid/app/build.gradle

```diff
@@ -1,21 +1,21 @@
-apply plugin: 'com.android.application'
+plugins {
+    alias(libs.plugins.android.application)
+}
 
 android {
-    compileSdkVersion 33
+    namespace = "de.ccc.events.postixdroid"
+
+    compileSdk 33
 
     defaultConfig {
-        applicationId "de.events.ccc.postixdroid"
+        applicationId "de.ccc.events.postixdroid"
```

Comment:
huh, I'm not sure how that happened. Should be the previous applicationid, or @pc-coholic?

Reply:
Phew.... Usually I would say "Do not touch" as to not disturb the update-process for existing installations.

But then again on the other hand, we do tend to install the app from scratch on all our rental devices and only the old (and hopefully soon to be permanently decomissioned) LCKJ-devices are getting the package through MDM (which I control)...

So I guess we could fix that oversight... @raphaelm @rixx any opinions on this?